src/cocaine/burlak/config.py

```diff
@@ -246,11 +268,23 @@ class Config(object):
                     'max': 2**16,
                     'required': False,
                 },
+                'post_interval': {
```

Comment:
According to the other code, it should be called `poll_interval` and why is it without `_sec`?